src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -47,42 +47,43 @@ export default class ResultTable extends React.Component {
     }
   }
 
-  cellRenderer = ({key, rowIndex, columnIndex, style}) => {
-    const {query, data: {data, meta}} = this.props;
-    const cols = this.getColumnList();
+  getCellRenderer = cols => {
+    return ({key, rowIndex, columnIndex, style}) => {
```

Comment:
nit: could be written as 

```
getCellRenderer = cols => ({key, rowIndex, columnIndex, style}) => {
  ...
```

Pretty common to write currying functions like this I think